lib/Net/GitHub/V3/Repos.pm

```diff
@@ -269,7 +269,7 @@ my %__methods = (
 
     # http://developer.github.com/v3/repos/contents/
     readme => { url => "/repos/%s/%s/readme" },
-    get_content => { url => "/repos/%s/%s/contents/%s" },
+    get_content => { url => "/repos/:owner/:repo/contents/:path", v => 2 },
```

Comment:
maybe we also want to list the valid optional arguments like by adding something like this `opts => [ qw{ref} ]`